### ISSUE TITLE
Checkstyle and IntelliJ import orders are not consistent

### ISSUE THREAD

**Issue by jeremyk-91** [palantir/atlasdb#1536]
Ran into this when updating `AtlasDbHttpClientsTest` in #1535.

Checkstyle:

```java
        <module name="ImportOrder"> <!-- Java Style Guide: Ordering and spacing -->
            <!-- <property name="groups" value="/.*/"/> -->
            <property name="groups" value="java,javax,org,com"/>
            <property name="option" value="top"/>
            <property name="sortStaticImportsAlphabetically" value="true"/>
        </module>
```

Intellij imports (idea.gradle)

```java
            importsTable.node.children().clear()
            ['true', 'false'].each { static1 ->
                ['java', 'javax', 'org', 'com', ''].each { name ->
                    importsTable.node.appendNode('package', [name: name, withSubpackages: 'true', static: static1])
                    importsTable.node.appendNode('emptyLine')
                }
            }
```

The problem comes if you have something like this:

```java
import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching; // (1)
import static org.hamcrest.Matchers.equalTo; // (2)
```

IntelliJ will format (2) before (1), but Checkstyle will complain since com comes before org. If you put (1) before (2) Checkstyle is happy. This is because of the sort static imports alphabetically rule - see [ImportOrderCheck](http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.html) for detail.

**Comment by rhero** [comment]
We're not going to get to this and it's not that annoying. Closing out.

**Comment by rhero** [comment]
Fixed in #1556.
